.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+on:
+  push:
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
```

Comment:
This can be condensed (although you may still prefer the name - several places we use it does not, I don't think), but can we make it an easier version to read? Something like https://github.com/panther-labs/test-automation/blob/main/.github/workflows/ci.yml#L10

Same comment for setting up Python.

Using the v2 or v3 vs what I believe is a commit SHA will be a lot easier to read. Unless you need those specific SHAs for some reason.

Reply:
Yea, I can -- I got into the habit of pinning to SHAs since it's one of GitHub's security hardening [recommendations](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions) for Actions.

That being said, we aren't really using any obscure Actions or handling sensitive data with these Workflows (and we aren't running on self-hosted runners).

Reply:
Addressed in [`f402406` (#223)](https://github.com/panther-labs/panther_analysis_tool/pull/223/commits/f40240698bc222edd382eb215d41f6c2676119f7).